koku/api/iam/serializers.py

```diff
@@ -95,7 +95,10 @@ def _create_default_preferences(user):
                     {'locale': settings.KOKU_DEFAULT_LOCALE}]
 
         for pref in defaults:
-            data = {'preference': pref, 'user': user}
+            data = {'preference': pref,
+                    'user': user,
+                    'name': list(pref.keys())[0],
+                    'description': 'default preference'}
```

Comment:
Maybe `'Default Preference'`, should we consider using `_()` so this is enabled for translation?